package.json

```
@@ -14,7 +14,9 @@
     "compile-styles": "sass --style=expanded ./src/index.scss dist/index.css",
     "css-minify": "cleancss --level 1 --format breaksWith=lf --output dist/index.min.css dist/index.css",
     "build": "npm run compile-styles && npm run css-minify",
-    "plop": "plop"
+    "plop": "plop",
+    "stylelint": "stylelint \"**/*.scss\"",
+    "stylelint-fix": "stylelint \"**/*.scss\" --fix"
```

nit-picking - consider the following (which is similar to the convention we have for "sub-scripts", like `test` and `test:without-build`
```suggestion
    "stylelint:fix": "stylelint \"**/*.scss\" --fix"
```